sdk/swagger.yaml

```diff
@@ -0,0 +1,720 @@
+          type: array
+          items:
+            $ref: "#/components/schemas/Expr"
+        object:
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/Expr"
+        builtin:
+          $ref: "#/components/schemas/ExprBuiltin"
+    EvaluatedExecutionContext:
+      type: object
+      properties:
+        properties:
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/Value"
+        schema: {}
+    Environment:
+      type: object
+      properties:
+        exprs:
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/Expr"
+        properties:
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/Value"
+        schema: {}
+        executionContext:
+          $ref: "#/components/schemas/EvaluatedExecutionContext"
+    CheckEnvironment:
+      type: object
+      allOf:     
+      - $ref: "#/components/schemas/Environment"
+      - $ref: "#/components/schemas/EnvironmentDiagnostics"
+    Trace:
+      type: object
+      properties:
+        def: 
+          $ref: "#/components/schemas/Range"
+        base:
+          $ref: "#/components/schemas/Value"
+    Value:
+      type: object
+      properties:
+        value: {}
+        secret:
+          type: boolean
+        unknown:
+          type: boolean
+        trace:
+          $ref: "#/components/schemas/Trace"
+      required:
+      - trace
+      - value
+    Pos:
+      type: object
+      properties:
+        line:
+          type: integer
+          description: "Line is the source code line where this position points. Lines are counted starting at 1 and incremented for each newline character encountered."
+        column:
+          type: integer
+          description: "Column is the source code column where this position points. Columns are counted in visual cells starting at 1, and are incremented roughly per grapheme cluster encountered."
+        byte:
+          type: integer
+          description: "Byte is the byte offset into the file where the indicated position begins."
+      required:
+      - line
+      - column
+      - byte
+    EnvironmentDiagnostic:
+      type: object
+      properties:
+        summary:
+          type: string
+        path:
+          type: string
+        range:
+          $ref: "#/components/schemas/Range"
+      additionalProperties:
+        type: object
+      required:
+        - summary
+    EnvironmentDiagnostics:
+      type: object
+      properties:
+        diagnostics:
+          type: array
+          items:
+            $ref: "#/components/schemas/EnvironmentDiagnostic"
+    EnvironmentDefinitionValues:
+      type: object
+      properties:
+        pulumiConfig:
+          type: object
+          additionalProperties: true
+        environmentVariables:
+          type: object
+          additionalProperties:
+            type: string

```

Comment:
Why are these present? If we're going to have them, we should also have `files`.

Reply:
I wondered if you would push back on that :-).  I feel they make the API easier to use, but am not married to them.  Happy to add files.